### PR TITLE
Fix live override

### PR DESCRIPTION
**Pull request by ehwhattaugonnado**
change exec live_override.cfg to exec MatchZy/live_override.cfg

### CODE REVIEW

**Review by shobhit-pathak — COMMENTED** [review]
Hey!

The plugin doesn't ship override cfgs as of now, so users can put it in the cfg folder itself.

I believe we should also have empty override cfg files, then making this path change will make sense.

**Comment by ehwhattaugonnado** [comment]
The changelog for 0.7.13 says "Simply create live_override.cfg and live_wingman_override.cfg in the cfg folder if you want to override any of the commands"  understood that to mean create the live_override.cfg in the same folder as live.cfg currently is in. 
I went ahead and added the empty override configs in the appropriate path.

File: cfg/MatchZy/live_wingman.cfg [hunk 1]
```diff
@@ -103,4 +103,4 @@ sv_voiceenable 1
 tv_relayvoice 0
 
 // execute override config
-exec live_wingman_override.cfg
\ No newline at end of file
+exec MatchZy/live_wingman_override.cfg
```
Comment (shobhit-pathak) [inline cfg/MatchZy/live_wingman.cfg:106]: Can you please make the same change in live.cfg as well?
Thanks!

Comment (ehwhattaugonnado) [reply cfg/MatchZy/live_wingman.cfg:106]: Done. I think I resynced to main and undid the initial change.

**Comment by borzaka** [comment]
It was a bad idea to add and empty override config. Because I already have one, and if I, and anybody who already has updates the plugin, it will overwrite.

File: cfg/MatchZy/ive_wingman_override.cfg [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (borzaka) [inline cfg/MatchZy/ive_wingman_override.cfg:1]: Typo in this file.

**Comment by borzaka** [comment]
Otherwise this fixes #222

**Pull request merged**